.github/workflows/merge-to-release-branch.yaml

```diff
@@ -0,0 +1,111 @@
+name: Merge to release branches
+
+on:
+  push:
+    branches:
+      - 'release-v**.x'
+
+env:
+  SDK_VERSION: "1.16.0"
+  GO111MODULE: on
+  K8S_VERSION: "1.19.2"
+  CONTAINER_RUNTIME: "docker"
+
+jobs:
+  release:
+    name: Release operator on Quay.io
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Git Repository
+        uses: actions/checkout@v2
+
+      - name: Set up PATH
+        run: |
+          mkdir -p $GITHUB_WORKSPACE/bin/
+          echo "PATH=$PATH:$GITHUB_WORKSPACE/bin/" >> $GITHUB_ENV
+
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: "^1.16"
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.7"
+          architecture: "x64"
+
```

Comment:
Do we need python for this job?  AFAICT the `release-operator` target doesn't need it.